File: Question 11/largest.js
//to find the largest odd number of the string.
function answer(num) {
    for (let i = num.length - 1; i >= 0; i--) {
    if (parseInt(num[i]) % 2 === 1) {
    return num.slice(0, i + 1);
    }
    }console.log(num);
    }

File: Question 11/largest.py
def answer(num): 
    for i in range(len(num) - 1, -1, -1):
        if int(num[i]) % 2 == 1:
            return num[:i + 1]
    
    return ""  # Return an empty string if no odd digit is found
    print(num)
